source/funkin/ui/freeplay/FreeplayState.hx

```diff
@@ -527,12 +528,15 @@ class FreeplayState extends MusicBeatSubState
       orangeBackShit.visible = true;
       alsoOrangeLOL.visible = true;
       grpTxtScrolls.visible = true;
+
+      // render optimisation
+      _parentState.persistentDraw = false;
```

Comment:
This is probably broken? Will need to test more thoroughly later.

Go into a song in Freeplay, exit to the Freeplay menu, then move back to the main menu. When this line is here you probably get a pink screen during the transition

Reply:
Tested this out, no pink screen. It's may also be that I'm missed something (?)

https://github.com/FunkinCrew/Funkin/assets/87835336/13e129d6-b639-467f-81ad-4bed9a8e05e9

Same test but exiting right after getting in Freeplay

https://github.com/FunkinCrew/Funkin/assets/87835336/5eb45a87-ff00-471f-b434-e09327312a52

Reply:
It did break something, from what I can see the sticker transition is not rendering

Reply:
I have used a transparent stickers mod for debugging "pink screen" issue, so there should not be any problems with sticker transition